components/mdx/Code.tsx

```diff
@@ -1,192 +1,169 @@
 // @ts-nocheck
 
 import React from "react";
-import Highlight, { defaultProps } from "prism-react-renderer";
-import { Prism } from "prism-react-renderer";
-import rangeParser from "parse-numeric-range";
-import classnames from "classnames";
+import Highlight, { Prism, defaultProps } from "prism-react-renderer";
+import classNames from "classnames";
 
 import FileIcon from "../icons/File";
 import CopyIcon from "../icons/Copy";
 import CopyButton from "../Copy";
-import classNames from "classnames";
 
 // Extend base classes
-(typeof global !== "undefined" ? global : window).Prism = Prism;
-require("prismjs/components/prism-docker");
+globalThis.Prism = Prism;
+import("prismjs/components/prism-docker");
 
 const diffBgColorMap = {
-  "+": "var(--prism-highlight-add)",
-  "-": "var(--prism-highlight-delete)",
-  "|": "var(--prism-highlight)",
+	"+": "var(--prism-highlight-add)",
+	"-": "var(--prism-highlight-delete)",
+	"|": "var(--prism-highlight)",
 };
 
 const SYMBOLS = {
-  normal: "|",
-  add: "+",
-  delete: "-",
+	normal: "|",
+	add: "+",
+	delete: "-",
 };
 
-function cleanTokens(tokens: any) {
-  const tokensLength = tokens.length;
+function cleanTokens(tokens: Token[][]): Token[][] {
+	const tokensLength = tokens.length;
+
+	if (tokensLength === 0) {
+		return tokens;
+	}
 
-  if (tokensLength === 0) {
-    return tokens;
-  }
+	const lastToken = tokens[tokensLength - 1];
 
-  const lastToken = tokens[tokensLength - 1];
+	if (lastToken.length === 1 && lastToken[0].empty) {
+		return tokens.slice(0, tokensLength - 1);
+	}
 
-  if (lastToken.length === 1 && lastToken[0].empty) {
-    return tokens.slice(0, tokensLength - 1);
-  }
+	return tokens;
+}
+
+interface InfoBarProps {
+	fileName?: string | null;
+	language: string;
+}
 
-  return tokens;
+function InfoBar({ fileName, language }: InfoBarProps) {
+	return (
+		<div className="flex mb-2 text-black dark:text-white text-sm font-bold">
+			{fileName == null ? null : (
```

Comment:
`Use '===' to compare with null` this should probably be turned off

Reply:
It do be its own rule, kinda thought it was lumped in to strict equality checking, but its not 😆